src/components/Button/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import style from './Button.st.css';
+import {ButtonNext} from 'wix-ui-core/button-next';
+import {ButtonProps as ButtonNextProps} from 'wix-ui-core/dist/src/components/button-next/button-next';
+
+export enum PRIORITY {
+  basic = 'basic',
+  primary = 'primary',
+  secondary = 'secondary'
+}
+
+export enum SIZE {
+  tiny = 'tiny',
+  small = 'small',
+  medium = 'medium',
+  large = 'large'
+}
+
+export interface ButtonProps extends ButtonNextProps {
+  priority?: PRIORITY;
+  size?: SIZE;
+}
+
+const CoreButton = ({priority, size, dataHook, ...rest}) =>
```

Comment:
@shlomitc i can catch the dataHook prop without declaring it on the api but i don't like this solution.
because the next dev' that will look at it will be puzzled